index.html

```diff
@@ -1,12 +1,30 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
+    <script src="todo.js"></script>
+    <link rel="stylesheet" href="todo.css">
```

Comment:
**put the script tag before the closing body tag**

> [Page Load Performance](https://betterprogramming.pub/improve-page-load-performance-with-these-different-script-loading-techniques-b0d912eae7b1)
>
> When you place the script tag at the bottom of the page after the main content, there will be some performance improvement. The content of the page will load and get parsed, as there is no blocking behaviour for rendering HTML since you have placed the script in the end.

Reply:
done